### PR TITLE
feat: add OpenAPIVersion component

### PR DESCRIPTION
**Pull request by kowalczyk-krzysztof**
Refs SWG-8477

### CODE REVIEW

**Comment by char0n** [comment]
Hi @kowalczyk-krzysztof,

Looks good and gets the job done. I was thinking about a possible simplification in https://github.com/swagger-api/swagger-ui/pull/9170

Now that we know we need to display OpenAPI version always, let's introduce a new `OpenAPIVersion` component instead of hacking `VersionStamp` which was intended originally just to display `Info Object`.`version` field. This way we'll achieve separation of concerns and allow `OpenAPIVersion` to be overriden by plugins, when somebody decides he/she wants to display the version in different way.

**Comment by kowalczyk-krzysztof** [comment]
I agree, a separate component seems better 👍🏻
I'll change my implementation.

**Comment by kowalczyk-krzysztof** [comment]
I changed my implementation to introduce `OpenAPIVersion` component.

**Comment by char0n** [comment]
Closing https://github.com/swagger-api/swagger-ui/pull/9170 as it's been incorporated in this PR.

File: src/core/containers/info.jsx [hunk 1]
```diff
@@ -19,14 +19,18 @@ export default class InfoContainer extends React.Component {
     const host = specSelectors.host()
     const externalDocs = specSelectors.externalDocs()
     const selectedServer = oas3Selectors.selectedServer()
+    const oasVersion = () => {
```
Comment (char0n) [inline src/core/containers/info.jsx:22]: You ideally don't want this here. In future we'll be separating non-plugin related code into plugins. This `info.jsx` component will be part of `oas2` plugin and it shouldn't know anything about other OpenAPI spec versions. Please have a look at https://github.com/swagger-api/swagger-ui/pull/9170/files how this is achieved.

Comment (kowalczyk-krzysztof) [reply src/core/containers/info.jsx]: Right. I updated the implementation and separated non-plugin related code.

**Review by char0n — APPROVED** [review]
Looks good! THanks!

**Pull request merged**